code/controllers/subsystems/vote.dm

```diff
@@ -199,10 +199,11 @@ var/datum/controller/subsystem/vote/SSvote
 						return 0
 		if(vote in choices)
 			if(current_votes[ckey])
-				choices[current_votes[ckey]]["votes"]--
+				choices[current_votes[ckey]["vote"]]["votes"] -= current_votes[ckey]["tally"]
 			voted += usr.ckey
-			choices[vote]["votes"]++	//check this
-			current_votes[ckey] = vote
+			var/vote_tally = (!ROUND_IS_STARTED || !(isnewplayer(usr) || isobserver(usr))) ? 1 : 0.5
```

Comment:
```suggestion
			var/vote_tally = (!ROUND_IS_STARTED || !(isnewplayer(usr) || !isobserver(usr))) ? 1 : 0.5
```
Otherwise doesn't this mean observers get a full vote?

Reply:
nah `IS NOT (isnewplayer(usr) OR isobserver(usr))`

Reply:
Oh whoops I missed the paren my bad